test/e2e/kb/standalone_test.go

```diff
@@ -68,7 +68,7 @@ func TestKibanaStandalone(t *testing.T) {
 	// set up a 1-node Kibana deployment manually connected to Elasticsearch
 	name := "test-kb-standalone"
 	esBuilder := elasticsearch.NewBuilder(name).
-		WithESMasterDataNodes(1, elasticsearch.DefaultResources).
+		WithESMasterDataNodes(2, elasticsearch.DefaultResources). // TODO .ds-metrics-fleet_server.agent_status-default index is created with a replica
```

Comment:
I'm not sure to understand the purpose of this TODO, and why `.ds-metrics-fleet_server.agent_status-default` is mentioned here (unless I'm missing something this tests only involves Kibana and Elasticsearch, not Agent?)

Reply:
Ya know, that's a very good question. From the diagnostics of the failure of this particular test:
cat_indices:
```
yellow open   .ds-metrics-fleet_server.agent_status-default-2023.12.18-000001 bS661iNXRBOHk8Qf6fs_2g   1   1         15            0    132.2kb        132.2kb false
```
Let me run locally and see what I can find.

Reply:
Verified locally that this index is being created with just kibana + elasticsearch:
```
yellow open .ds-metrics-fleet_server.agent_status-default-2023.12.19-000001 Lpe5MTJ8R6CNusmbk6gwrw 1 1 6 0 52.9kb 52.9kb 52.9kb
```

Will reach out to Kibana team.

Reply:
I fixed a couple of these (seems like maybe not all?) Thibault opened an issue https://github.com/elastic/cloud-on-k8s/issues/7376 which has a reference to an issue I opened for fleet. I hope this is helpful.

Reply:
I verified with the Fleet/Kibana team, fleet integration is now enabled by default, so this index will exist with just vanilla Kibana+Elasticsearch. I'll update the comment/todo to be more clear. We still should be able to control num replicas sometime in the future and go back down to 0 replicas.